__tests__/getEndpoints.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET endpoints", () => {
+  describe("GET /api/topics", () => {
+    test("200: responds with a 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("200: responds with array of topic objects with correct properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((res) => {
```

Comment:
We should add an assertion to check the length of the array when asserting inside a for each because the current test would pass with an empty array 🙂